docs/developers/project-overview.md

```diff
@@ -0,0 +1,3 @@
+# Project Overview
+
+This doc will give an overview of the project. Project vision, etc.
```

Comment:
@dakotabenjamin thinking that section is not needed as the READ.ME file should give that project overview.